Mathlib/Data/Set/Finite.lean

```diff
@@ -1394,9 +1394,10 @@ theorem not_injOn_infinite_finite_image {f : α → β} {s : Set α} (h_inf : s.
   have h := not_injective_infinite_finite
             ((f '' s).codRestrict (s.restrict f) fun x => ⟨x, x.property, rfl⟩)
   contrapose! h
+  --  re-porting note: the porting note below can probably be removed now: there was an extra
+  --  `rwa [not_not] at h` afte the next `rw`.
   --porting note: why do we have `contrapose!` if the `push_neg` behaviour doesn't work?
-  rw [injective_codRestrict, ← injOn_iff_injective]
-  rwa [not_not] at h
+  rwa [injective_codRestrict, ← injOn_iff_injective]
```

Comment:
I would probably remove the old porting note, but wanted to give visibility to this, in case someone objects.